### DISCUSSION TITLE
don't get Alexa connected to emulated Hue

### DISCUSSION THREAD

**Discussion by thorschtn** [hass-emulated-hue/core#122] [Q&A] [answered]
Hi!
I don't get Alexa connected to emulated Hue.

System environment:
- HA Supervised core-2021.2.3
- supervisor-2021.02.11
- hass-emulated-hue 0.2.3 (tried master and dev)

hass-emulated-hue starts without error messages on both ports:
2021-02-23 10: 44: 08,540 INFO emulated_hue.api - Started HTTP webserver on port 80
2021-02-23 10: 44: 08,558 INFO emulated_hue.api - Started HTTPS webserver on port 443
However, I only get an answer from port 80 and not from port 443, is that correct?

When I search with the Hue app (iPhone 11), I find the Hue emulated bridge and the connected devices and can also switch them.

The Alexa app, however, does not find a bridge (neither when I search for V1 Bridge, nor for V2 Bridge). There is not even a pairing request in the ha notifications).

What am I doing wrong?

**Comment by alexyao2015** [comment, marked as answer]
Please refer to https://github.com/hass-emulated-hue/core/discussions/90
